metrics/heapster.go

```diff
@@ -326,7 +326,7 @@ func getPodLister(kubeClient *kube_client.Clientset) (v1listers.PodLister, error
 
 func validateFlags(opt *options.HeapsterRunOptions) error {
 	if opt.MetricResolution < 5*time.Second {
-		return fmt.Errorf("metric resolution needs to be greater than 5 seconds - %d", opt.MetricResolution)
+		return fmt.Errorf("metric resolution could not be less than 5 seconds - %d", opt.MetricResolution)
```

Comment:
Use `should not`, `cannot`, or `must not`.  `could not` implies (in this particular case) that it's no longer true, because of the past tense.